fs/proc.c

```diff
@@ -101,7 +101,7 @@ static int proc_refresh_data(struct fd *fd) {
     struct proc_entry entry = fd->proc.entry;
     ssize_t size = entry.meta->show(&entry, fd->proc.data);
     if (size < 0)
-        return size;
+        return (int)size;
```

Comment:
Why add this?

Reply:
Xcode warns about implicit cast from ssize_t to smaller int (loses precision blah blah), I've added explicit cast to silence it, but didn't mean to add it to commit nor push it.

I've noticed when I've created PR, but it was not worth it to stash, switch back to proc-short-read branch, remove it, amend commit, force push and switch back to what I was working on.

Should I remove it?

Reply:
For some reason I haven't seen this warning before. I'd say it's fine then.